src/cqlEditorWithTerminology/CqlEditorWithTerminology.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import MadieAceEditor, { EditorPropsType } from "../AceEditor/madie-ace-editor";
+import { Allotment } from "allotment";
+import "allotment/dist/style.css";
+import "./CqlEditorWithTerminology.scss";
+import RightPanel from "../rightPanel/RightPanel";
+
+const CqlEditorWithTerminology = ({
+  value,
+  onChange,
+  height,
+  parseDebounceTime = 1500,
+  inboundAnnotations,
+  inboundErrorMarkers,
+  readOnly = false,
+  validationsEnabled = true,
+  setOutboundAnnotations,
+}: EditorPropsType) => {
+  return (
+    <div className="allotment-wrapper">
+      <Allotment defaultSizes={[175, 125]} vertical={false}>
+        <Allotment.Pane>
+          <div style={{ borderWidth: "24px", borderColor: "#ededed" }} />
+          <div className="left-panel">
+            <div className="panel-content">
+              <MadieAceEditor
+                value={value}
+                onChange={onChange}
+                height={height}
+                parseDebounceTime={parseDebounceTime}
+                inboundAnnotations={inboundAnnotations}
+                inboundErrorMarkers={inboundErrorMarkers}
+                readOnly={readOnly}
+                validationsEnabled={validationsEnabled}
+                setOutboundAnnotations={setOutboundAnnotations}
+              />
+            </div>
+          </div>
+        </Allotment.Pane>
+        <Allotment.Pane>
+          <RightPanel />
```

Comment:
can we name this bit more meaningful.

Reply:
Sure, I have changed the name to TerminologySectionPanel